jazz-build-module/environment-deployment-metadata-loader.groovy

```diff
@@ -1,562 +0,0 @@
-#!groovy?
```

Comment:
you are deleting this file which is not what we want. Remove this file from your PR